runtimes/pythonrt/py-sdk/autokitteh/google.py

```diff
@@ -260,6 +260,6 @@ def google_id(url: str) -> str:
     """
     match = re.match(r"(.*/d/(e/)?)?([\w-]{20,})", url)
     if match:
-        return match.group(3)
+        return match.group(2)
```

Comment:
Undo this change - the previous version is the right one

```suggestion
        return match.group(3)
```

Reply:
so 2 or 3 ? what's the correct value ?

Reply:
3 is correct. I have a task to add a unit test for this, but too many urgent things before that.